engine/class_modules/monk/sc_monk.cpp

```diff
@@ -171,6 +173,15 @@ void monk_action_t<Base>::apply_buff_effects()
   parse_effects( p()->buff.press_the_advantage );
 
   // Conduit of the Celestials
+  parse_effects( p()->buff.flight_of_the_red_crane );
+  parse_effects( p()->buff.heart_of_the_jade_serpent_cdr );
+  parse_effects( p()->buff.heart_of_the_jade_serpent_cdr_celestial );
+  parse_effects( p()->buff.inner_compass_crane_stance );
+  parse_effects( p()->buff.inner_compass_ox_stance );
+  parse_effects( p()->buff.inner_compass_serpent_stance );
+  parse_effects( p()->buff.inner_compass_tiger_stance );
```

Comment:
Inner Compass doesn't modify actions.
Belongs in monk_t::parse_player_effects()